setup.py

```diff
@@ -10,10 +10,12 @@
             'allennlp',
```

Comment:
To add package versions especially for `pandas` to not have issues when loading/saving pickled data

Reply:
fixed